packages/plugin-apex/test/commands/force/apex/test/run.test.ts

```diff
@@ -1208,4 +1208,29 @@ describe('force:apex:test:run', () => {
     .it('should set exit code as 0 for passing run', () => {
       expect(process.exitCode).to.eql(0);
     });
+
+
+
+  test
+    .withOrg({ username: TEST_USERNAME }, true)
+    .loadConfig({
+      root: __dirname
+    })
+    .stub(process, 'cwd', () => projectPath)
+    .stub(TestService.prototype, 'runTestAsynchronous', () => testRunSimple)
+    .stdout()
+    .command([
+      'force:apex:test:run',
+      '--tests',
+      'MyApexTests',
+      '--wait',
+      '20'
+    ])
+    .it('should pass and return human-readable results when the wait argument is passed', ctx => {
```

Comment:
Can we add one more test for the scenario where the wait flag is passed and the result format is not set as human-readable, maybe junit or json? Also @AnanyaJha if the --json flag is passed, should it always output results in json format? That came up in #236.

Reply:
done

Reply:
Great point randi, if the `--json` flag is passed we should be displaying results in that format rather than a default human-readable format.

Reply:
Have you been able to run the tests against this PR? It doesn't look like they're running in the PR for some reason

Reply:
Only locally.  I noticed there are few jobs with "Waiting for status to be reported" and was meaning to ask about that.  I haven't seen this before.  Could it be a permissions issue?  Or perhaps a step I'm missing when posting a PR.